tactics/tactics.ml

```diff
@@ -83,6 +83,14 @@ let _ =
 
 let apply_solve_class_goals = ref false
 
+let _ =
+  declare_bool_option
+    { optdepr = true; (* remove in 8.8 *)
+      optname = "Perform typeclass resolution on apply-generated subgoals.";
+      optkey = ["Typeclasses";"Resolution";"After";"Apply"];
```

Comment:
First of all, not sure why we're adding an option here.
Second, it should be called "Typeclass Resolution After Apply".

Reply:
In fact, the option is added for compatibility (it already existed as such but was documented as "Typeclass Resolution After Apply".